templates/dashboard/includes/_orders_table.html

```diff
@@ -57,7 +58,11 @@
           </td>
           {% trans "Guest" context "Anonymous user account value" as guest %}
           <td>
-            {{ order.user|default:guest }}
+            {% if order.user %}
+              {{ order.user|obfuscate_string }}
```

Comment:
If customer filled last name and first name, should we leave the first letter only, or first from both words?

Reply:
We've been discussing this internally with @elwoodxblues and settled that displaying name and surname isn't really an issue, its when you have extra identification like e-mail, living address or phone that it turns into an issue, but we'll be filtering out those anyway.

On sidenote, the unfiltered`{{ order.user }}` prints user email here, so